noise.js

```diff
@@ -1,3 +1,33 @@
-$(document).ready( function() {
-  // your code here
+$(document).ready(() => {
+
+  $('.instrument').on('click', 'button', function click() {
+    const note = $(this)[0].innerHTML;
+    $(`#${note}Audio`)[0].load();
+    $(`#${note}Audio`)[0].play();
+
+    $('body').keydown((event) => {
+      if (event.keyCode === 67) {
+        $('#cAudio')[0].load();
```

Comment:
I think you should be able to load the audio once, and then use it with play on each click.

It also might be more compact doing this with a loop instead of an if-else.